css/openmicroscopy.css

```diff
@@ -591,6 +591,16 @@ p.job-title {
     padding-left: 5px;       
 }
 
+.citation {
+    border-radius: 2px;
+    background-color: #e3f2fd;
+    font-style: italic;
+    margin-bottom: 10px;
+    padding: 10px 15px;
+}
+
+a.pmid::before { content: "PMID: "; }
+
 /* ========== page: Community ==========*/
```

Comment:
@lunson: the only reason for which this PR conflicts with #103 is that the former PR modifies the line underneath. Would it be possible to move the added items right under the `section: About` line? That way I would expect both set of CSS changes would merge cleanly and we could deploy both the citation and the security changes on staging.